advanced_source/static_quantization_tutorial.rst

```diff
@@ -0,0 +1,675 @@
+(beta) Static Quantization with Eager Mode in PyTorch 
+========================================================= 
+**Author**: `Raghuraman Krishnamoorthi <https://github.com/raghuramank100>`_  
```

Comment:
Please add your name to the list of authors

Reply:
sure, I added myself to "edited by"